std/encoding.d

```diff
@@ -1355,6 +1349,7 @@ bool isValid(E)(const(E)[] s)
     return s.length == validLength(s);
 }
 
+///
 unittest
 {
     assert(isValid("\u20AC100"));
```

Comment:
This unittest looks too scanty to be a ddoc'd example. It looks like it's just to prevent regression of a fixed bug. Do we really need this in the docs?